src/order/closure.lean

```diff
@@ -83,6 +83,18 @@ lemma closure_union_closure_le {α : Type u} [semilattice_sup α] (c : closure_o
   c x ⊔ c y ≤ c (x ⊔ y) :=
 sup_le (c.monotone le_sup_left) (c.monotone le_sup_right)
 
+/--
+Every Galois connection induces a closure operator given by the composition. This is the partial
+order version of the statement that every adjunction induces a monad.
+-/
+def closure_operator_of_galois_connection {β : Type u} [preorder β]
```

Comment:
Let's move this outside the `closure_operator` namespace and call it `galois_connection.closure_operator`...